src/SearchAnime.js

```diff
@@ -0,0 +1,47 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+
+
+const SearchedAnime = () => {
+
+    const { animeSearched } = useParams();
+    const [anime, setAnime] = useState([]);
+
+    useEffect( () => {
+        axios({
+            url: 'https://api.jikan.moe/v4/anime',
+            method: 'GET',
+            dataResponse: 'json',
+            params: {
+                q: animeSearched,
+            }
+        }).then( (res) => {
+            setAnime(res.data.data);
+        })
```

Comment:
Try adding a `.catch()` for alerting the user if there are no animes with this name maybe? I tried searching "agsagsagh" and there was nothing telling me there were no results